packages/utils/src/queue.ts

```diff
@@ -17,7 +14,7 @@ export class AsyncQueue {
   // Awaits the finish of all promises that have been queued up before it
   // And will expire itself (reject the promise) after waiting limit ms
   // or never expire, if limit is not set
-  public async addToQueue<T = any>(promiseGenerator: () => Promise<T>, limitInMs = 0): Promise<T> {
+  public async addToQueue<T = any>(promiseGenerator: () => Promise<T>): Promise<T> {
```

Comment:
I think semver purists would call this a breaking change 🤔 @jooohhn strong thoughts here? we could keep this and mention that it's deprecated, though since no one has listed our utils as a dependency the chance that this would upset someone is low

Reply:
I think we're about ripe to have `2.0.0` in collecting a bunch of deprecated code pieces though - maybe we should also discuss a strategy about going about that 😄

Reply:
That raises an interesting question- do we consider all the code in this repo to be public-facing? I wouldn't have thought that anyone would go digging into the internals of our logic for queueing and sending requests.